UnitySDK/Assets/ML-Agents/Scripts/Timer.cs

```diff
@@ -75,6 +82,12 @@ public double TotalSeconds
             set {}  // Serialization needs this, but unused.
         }
 
+        public Dictionary<string, GaugeNode> Gauges
+        {
+            get { return m_Gauges; }
+            set { m_Gauges = value; }
```

Comment:
I left the setter empty in some of the other properties; unfortunately serialization seems to need them, but we don't really want them settable, and Rider complains about the unused `value` parameter if they're unimplemented.

I don't really care either way, but we probably should be consistent.

Reply:
took your suggestion.